components/tools/OmeroJava/test/integration/BfPixelBufferTest.java

```diff
@@ -55,7 +52,6 @@ private void setUpTestFile(String fileName) throws Throwable,
     }
 
     private void tidyUp() throws Throwable {
-        FileUtils.deleteQuietly(destPath);
```

Comment:
Do we just no longer need to clean these temporary files up at all?

Reply:
The variable was never used. So we had a NPE without knowing

Reply:
Aha! Heh, ta.